lib/transport/data_validation/data_validation.ex

```diff
@@ -28,4 +30,28 @@ defmodule Transport.DataValidation do
       {:error, error} -> {:error, error}
     end
   end
+
+  @doc """
+  Finds a feed source.
+  """
+  @spec find_feed_source(map()) :: {:ok, FeedSource.t} | {:error, any()}
+  def find_feed_source(%{} = params) do
+    params
+    |> FindFeedSource.new
+    |> Project.execute
+  end
+
+  @doc """
+  Creates a feed source.
+  """
+  @spec create_feed_source(map()) :: {:ok, FeedSource.t} | {:error, any()}
+  def create_feed_source(%{} = params) do
+    params
+    |> CreateFeedSource.new
+    |> CreateFeedSource.validate
+    |> case do
+      {:ok, command} -> Project.execute(command)
```

Comment:
You want `FeedSource.execute`

Reply:
It could, though for now it is the project who finds and creates feed sources (basically for scoping).